src/main/resources/sql/init.sql

```diff
@@ -0,0 +1,46 @@
+create table if not exists tb_menu
```

Comment:
You may add some code like 'create database if not exist...'.
Did you verify locally? If not, please do it.